--- projectM-complete-2.1.0-Source/src/libprojectM/CMakeLists.txt~	2012-05-20 05:43:26.000000000 +0200
+++ projectM-complete-2.1.0-Source/src/libprojectM/CMakeLists.txt	2017-03-12 00:27:28.921676820 +0100
@@ -34,6 +34,7 @@
 #
 
 ADD_DEFINITIONS(-DCMAKE_INSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}")
+ADD_DEFINITIONS(-DCMAKE_LIB_DIR="${CMAKE_LIB_DIR}")
 
 SET(LIB_INSTALL_DIR "\${exec_prefix}/lib"
   CACHE PATH "The subdirectory relative to the install prefix where libraries will be installed (default is /lib)"
--- projectM-complete-2.1.0-Source/src/libprojectM/libprojectM.pc.in~	2012-05-20 05:43:26.000000000 +0200
+++ projectM-complete-2.1.0-Source/src/libprojectM/libprojectM.pc.in	2017-03-12 00:28:04.319482765 +0100
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@LIB_INSTALL_DIR@
+libdir=@CMAKE_LIB_DIR@
 includedir=@CMAKE_INSTALL_PREFIX@/include
 pkgdatadir=@CMAKE_INSTALL_PREFIX@/@RESOURCE_PREFIX@
 sysconfdir=@CMAKE_INSTALL_PREFIX@/@RESOURCE_PREFIX@
